src/modules/layout/components/TransparentHeader.js

```diff
@@ -23,19 +23,24 @@ const HorizontalLeft = styled.View`
   align-items: center;
 `
 
-const BoxShadow: StyledComponent<{}, ThemeType, *> = styled.View`
+const BoxShadow: StyledComponent<{float: boolean}, ThemeType, *> = styled.View`
   background-color: transparent;
-  position: absolute;
-  z-index: 100;
-  top: 0;
-  left: 0;
-  right: 0;
   height: ${props => props.theme.dimensions.modalHeaderHeight};
+  ${props => props.float
```

Comment:
Why is this not needed if a menu is shown? Maybe we should just have one prop `withMenu` and remove the prop `float`. I think the property `float` would require documentation.

Reply:
Imo these two props are not connected, only in this specific case. Am I understanding correctly that you would use only one prop?